backend/node/src/middleware/validate.ts

```
@@ -9,19 +9,32 @@ interface Schema {
-      }
-      if (schema.query) {
-        req.query = schema.query.parse(req.query) as Record<string, string>;
+      // 👉 Ako je predan običan ZodType (npr. idSchema)
+      if (
+        'parse' in schema &&
+        !('body' in schema || 'params' in schema || 'query' in schema)
+      ) {
+        // automatski pretpostavi da validira parametre
+        req.params = (schema as ZodType).parse(req.params) as Record<string, string>;
+        return next();
       }
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)